server/sonar-server/src/test/java/org/sonar/server/issue/notification/DoNotFixNotificationDispatcherTest.java

```diff
@@ -84,7 +85,7 @@ public void ignore_other_resolutions() {
     Multimap<String, NotificationChannel> recipients = HashMultimap.create();
```

Comment:
lines 85 to 88 are actually useless to this test.
when resolution is not supported, `findSubscribedRecipientsForDispatcher` is not even called

(I tried and verified locally)